snippets/header-drawer.liquid

```diff
@@ -21,7 +21,22 @@
         <div class="menu-drawer__navigation-container">
           <nav class="menu-drawer__navigation">
             <ul class="menu-drawer__menu has-submenu list-menu" role="list">
-              {%- for link in section.settings.menu.links -%}
+                  {% assign showB2BNav = false %}
```

Comment:
Is the expectation that people will change this to true by editing the theme if they want the b2b nav?

Reply:
Yes